packages/grafana-toolkit/package.json

```diff
@@ -78,5 +83,8 @@
   "devDependencies": {
     "@types/glob": "^7.1.1",
     "@types/prettier": "^1.16.4"
+  },
+  "_moduleAliases": {
```

Comment:
What is this needed for?

Reply:
no idea -- it was in the existing 🤷‍♂

Reply:
the npm package `puppeteer` split a while ago into `puppeteer-core` and friends. This was needed for backward compat of packages that referred to it as `puppeteer`. Not sure if this is still needed.